blog/aep-2026-release.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Announcing the aep-2026 Release
+date: 2025-12-08
+authors:
+  - name: Marsh Gardiner
+  - name: Yusuke Tsutsumi
+  - name: Alex Stephen
+---
+
+import { Image } from 'astro:assets';
+import aep300Image from './aep_300_image.png';
+
+# The Coordination Challenge
+
+Building software is a team sport, but keeping everyone on the same page is
+hard. When every team has to invent their own API patterns — deciding how
+pagination works, or how to format errors — we end up with a lot of wasted
+effort and systems that don't quite fit together. This isn't just about having
+"standards". It's about ensuring that users and tools don't have to
+continuously relearn the same standards.
+
+# A Shared Standard and Tools
+
+The aep-2026 edition is our answer to that friction. It is an opinionated API
+design specification, providing well-defined requirements around common REST
+concepts like CRUD standard methods, pagination, filtering, and more. By
+agreeing on these common patterns, we make it easier for services to talk to
+each other and for developers to move between projects without needing to learn
+idiosyncratic conventions.
+
+<Image
+  src={aep300Image}
+  alt="Screenshot of the AEP-300 rule describing edition-based versioning"
+/>
+
+AEP is an open-source, "anti-bike shedding" kit. It answers the routine
+questions so we can all focus on the interesting ones. It’s about a shared
+foundation that lifts everyone up and lets everyone build on the same tooling
+platform.
+
+# Stability for Builders
+
+To build lasting tools, you need a solid foundation. You can't build tooling on
+top of a spec that changes regularly. The aep-2026 edition is a major milestone
+because we are freezing the core specification and establishing a release
+cadence with [AEP-300](https://aep.dev/300/). This creates a stable platform
+for the community:
```

Comment:
```suggestion
because we are introducing a stable version of the specification and establishing a release
cadence with [AEP-300](https://aep.dev/300/). This decouples stability for consumers and evolving our best practices:
```